docs/content-onboarding/onboarding.md

```diff
@@ -110,24 +112,29 @@ Host github.com
  IdentityFile ~/.ssh/id_ed25519
 ```
 
-* save and close the file (**without** a file extension). Make sure to use "UTF-8" encoding system when editing the file, this can be done using Notepad++ or in Visual Studio click on the encoding system right bottom > Save with encoding > UTF-8 . 
+* Save and close the file (**without** any file extension). Make sure to use "UTF-8" encoding system when editing the file, this can be done using Notepad++ or in Visual Studio click on the encoding system right bottom > Save with encoding > UTF-8 . 
+ *Hint:*
+ It is recommended to use [Notepad++](https://notepad-plus-plus.org/), [Visual Studio Code](https://code.visualstudio.com/), or Notepad to view the files. The files contain both the key and the key title.
 
-!!! hint
-    1. Prefer [Notepad++](https://notepad-plus-plus.org/), [Visual Studio Code](https://code.visualstudio.com/), or Notepad to view the files. The files contain both the key and the key title. 
-    2. Next login to Github > setting > SSH and GPG keys > New SSH key. Make sure to paste the key under key section and title under title section. (Look for steps under the 'Deploy SSH Public Key to GitHub' heading below.)
+* Next, log in to your GitHub account and go to:  
+   *Settings* → *SSH and GPG keys* → *New SSH key*
+    - Paste the SSH key into the *Key* field.  
+    - Enter a descriptive title in the Title field (e.g., "ssh-key-TUC")
+       
+(For detailed steps, refer to the section *Deploy SSH Public Key to GitHub* below)
+       
+**Note:** You can ignore the message:  *"Before you can add an SSH key, you need to generate it."*  — You have already created your SSH key.
 
-**Ignore** the statement "Before you can add an SSH key you need to generate it." You just created this key.
 
 
 ### Linux (works for MacOS as well)
 
-
 #### Generate SSH keys
 
-* Create a public/private key pair using the ssh-keygen commandline tool.  
+* Create a public/private key pair using the `ssh-keygen` command-line tool. 
```

Comment:
Small note: Lines should end with exactly 0 or 2 spaces. (0 spaces = continue the line, 2 spaces = enforce line break)